### DISCUSSION TITLE
MessageBar in FluentUI v9 missing

### DISCUSSION THREAD

**Discussion by sw00sh0r** [microsoft/fluentui#27786] [General] [answered]
Hi,

I am currently migrating from v8 to v9 and noticed that the MessageBar component is missing in V9.
Are there any plans to migrate this one?
Unfortunately I couldn't find anything about it in the roadmap.

Thanks & kind regards.

**Comment by sgbdev** [comment, marked as answer]
I think "Alert" is targeted as the replacement for MessageBar. https://react.fluentui.dev/?path=/docs/preview-components-alert--default

**Reply by adem-pointsharp** [reply]
Are you sure? According to this comment, Alert will be deprecated and MessageBar will come back: https://github.com/microsoft/fluentui/issues/27949#issuecomment-1559409141

**Comment by sw00sh0r** [comment]
Actually. I swear I just checked the alert a few days ago and it looked completely different.
Thank you very much for the advice!
